### ISSUE TITLE
(graphviz) Install/update failure

### ISSUE THREAD

**Issue by ebbek** [chocolatey-community/chocolatey-packages#1745]
Installation attempts to install the 32-bit version of Graphviz on a 64-bit system and fails.

## Expected Behavior

Graphviz should be updated or installed without errors.

## Current Behavior

When attempting an update or a re-install I get the following error message:

```
Graphviz v2.49.3 [Approved]
graphviz package files install completed. Performing other installation steps.
The package Graphviz wants to run 'chocolateyInstall.ps1'.
Note: If you don't run this script, the installation will fail.
Note: To confirm automatically next time, use '-y' or consider:
choco feature enable -n allowGlobalConfirmation
Do you want to run the script?([Y]es/[N]o/[P]rint): y

ERROR: This package does not support 64 bit architecture.
The install of graphviz was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\Graphviz\tools\chocolateyInstall.ps1'.
 See log for details.
```

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Package Version used: 2.49.3
* Operating System and version:  Windows 10 [Version 10.0.19044.1348]
* Chocolatey version: 0.11.3
* Install/uninstall gist:

**Comment by AdmiringWorm** [comment]
Not able to reproduce it trying to install 32bit with a normal call to `choco install`.

It is possible that 32bit installation was remembered in the past for the package (maybe because of a bug that was in 0.10.15, which was fixed in 0.11.0 but requires an uninstall then a new install of the package).

Without a reproduction I the issue and/or a link to the log gist it is not possible for me to say for sure, as such please update the issue with the steps to reproduce the error and upload the chocolatey log to a gist and link to it in the issue.

**Comment by AdmiringWorm** [comment]
As the issue have not been updated with reproduction, and/or a chocolatey log, I will be closing this issue now.

Feel free for it to be re-opened once the issue description has been updated with a reproduction or the actual chocolatey log.
